stampery-trailbot-ubuntu/azuredeploy.json

```diff
@@ -32,7 +32,7 @@
         "description": "Client public key. Please install Trailbot Client in your personal computer, open it and it will generate a key you must paste here."
       }
     },
-    "vmSize": {
+    "size": {
```

Comment:
sorry - to be clear, remove the param altogether and just use the var you have below... set a value appropriate for your workload (i,.e. the default you have here)